### DISCUSSION TITLE
NV upgrade process for PreCommitSectorBatch2?

### DISCUSSION THREAD

**Discussion by Fatman13** [filecoin-project/FIPs#826] [General]
Why a new message like [PreCommitSectorBatch2](https://filecoinproject.slack.com/archives/C027TQMUVJN/p1694204364685999) was added to the protocol without it going through the FIP process? 

Shouldn't protocol change like this be governed by FIP / [nv upgrade](https://github.com/filecoin-project/core-devs/discussions/140#discussion-5202703) process? Or maybe there is already an FIP that I have missed? Thank you!

![image](https://github.com/filecoin-project/FIPs/assets/1591330/1b1bddeb-a618-4b02-b1bd-93f6b4bdc7bc)

**Comment by jennijuju** [comment]
If you do a quick search in FIP repository - you will find PreCommitSectorBatch2 was introduced in https://github.com/filecoin-project/FIPs/blob/8faf1e17c3a075286460fca3901b5bb1337299ac/FIPS/fip-0041.md?plain=1#L34

**Reply by Fatman13** [reply]
I see. Thanks for sharing.
